frontend/src/features/admin-form/responses/IndividualResponsePage/IndividualResponseNavbar.tsx

```diff
@@ -130,10 +154,31 @@ export const IndividualResponseNavbar = (): JSX.Element => {
       </Flex>
       <Flex gridArea="respondent" justify="center" align="center">
         <Skeleton isLoaded={!isLoading}>
-          <Text textStyle="h2" as="h2">
-            {t('features.common.response')}
-            {currentResponseNumber ? ` #${currentResponseNumber}` : ''}
-          </Text>
+          <Stack direction="row" justify="center" align="center">
+            <Text textStyle="h2" as="h2">
+              {t('features.common.response')}
+              {currentResponseNumber ? ` #${currentResponseNumber}` : ''}
+            </Text>
+            {isAdminPrintPdfEnabled && (
+              <Box>
+                <IconButton
+                  aria-label="Print"
+                  icon={<FaRegFilePdf />}
+                  onClick={() => {
+                    datadogRum.addAction('individualresponsepage.navbar.print', {
+                      meta: {
```

Comment:
Not an issue, but i dont see how this can be used. No issue tracking this either since it should not cost us more.